snapcraft/tests/test_pluginhandler.py

```diff
@@ -987,17 +956,16 @@ def setUp(self):
         self.get_build_properties_mock.return_value = []
         self.addCleanup(patcher.stop)
 
-        part_name = 'test_part'
-        self.handler = pluginhandler.load_plugin(part_name=part_name,
-                                                 plugin_name='nil')
+        self.handler = _load_plugin('test_part')
+
         self.handler.makedirs()
 
     def test_mark_done_clears_later_steps(self):
         for index, step in enumerate(common.COMMAND_ORDER):
-            shutil.rmtree(self.parts_dir)
+            with contextlib.suppress(FileNotFoundError):
+                shutil.rmtree(self.parts_dir)
```

Comment:
Yeah but that would ignore permissions errors etc. as well, I assume? I'd want to see those errors.